libr/core/cconfig.c

```diff
@@ -3970,7 +3970,11 @@ R_API int r_core_config_init(RCore *core) {
 	{
 		char *p = r_file_path ("git");
 		bool found = (p && (*p == '/' || *p == 'g'));
-		SETBPREF ("vc.rvc", r_str_bool (!found), "Use RVC instead of git (not recommended)");
+		if (found) {
+			SETBPREF ("prj.vc.type", "git", "What should projects use as a vc");
+		} else {
+			SETBPREF ("prj.vc.type", "rvc", "What should projects use as a vc");
+		}
```

Comment:
Set a callback to check if the value set is valid or not and return a list of them when value is ?.

other config vars do that see asm.bits or asm.os as example